ArcGISRuntimeSDKQt_CppSamples/Layers/DisplayOgcApiCollection/README.md

```diff
@@ -0,0 +1,37 @@
+# Display OGC API Feature Collection
+
+This sample demonstrates how to display an OGC API feature collection and query features while navigating the map view.
+
+![](screenshot.png)
+
+## Use case
+
+When panning the map view, it may be necessary to query the OGC API feature table for additional features within the new visible extent.
```

Comment:
May be? 

The sample is using `FeatureRequestMode::ManualCache` so it will be required. Consider clarifying.

Reply:
I spoke with the samples team about this and we'll update the text